### PR TITLE
Rename marbles_transfer to asset-transfer-secured-agreement

### PR DESCRIPTION
**Pull request by stephyee**
- Rename marbles_transfer to a more generic and streamlined asset
- Move sample to new directory
- Remove TODO for marbles by size query

### CODE REVIEW

File: asset-transfer-secured-agreement/README.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+
+## Deploy the chaincode
+
+Now that we can operate as both organizations, we need install the asset transfer smart contract on the peers of Org1 and Org2, and deploy the chaincode to the channel approving and committing the chaincode definition.
+
+### Install and approve the chaincode as Org1
+
+Open the Org1 terminal. Run the following command to package the assets transfer chaincode:
+```
+peer lifecycle chaincode package assets_transfer.tar.gz --path  ../chaincode/assets_transfer --lang golang --label assets_transfer_1
```
Comment (denyeart) [inline asset-transfer-secured-agreement/README.md:121]: Directory needs to change

File: asset-transfer-secured-agreement/README.md [hunk 2]
```diff
@@ -0,0 +1,342 @@
+# assets private asset transfer scenario
```
Comment (nikhil550) [inline asset-transfer-secured-agreement/README.md:1]: This is an awkward title. The phrase assets transfer is also awkward. I would rename the title, secure private asset transfer scenario, and call it the secure asset transfer smart contract.

File: asset-transfer-secured-agreement/README.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+
+The assets transfer smart contract demonstrates how an asset can be represented and traded between organizations in a Hyperledger Fabric blockchain channel, while keeping details of the asset and transaction private.
+Each on-chain asset is a non-fungible token (NFT) that represents a specific asset having certain immutable metadata properties (such as size and color) with a unique owner. When the owner wants to sell the asset, both parties need to agree to the same price before the asset is transferred. The assets transfer smart contract enforces that only the owner of the asset can transfer the asset. In the course of this tutorial, you will learn how Fabric features such as state based endorsement, private data, and access control come together to provide secured transactions that are both private and verifiable.
+
+## Scenario requirements
+
+The assets transfer scenario is bound by the following requirements:
```
Comment (nikhil550) [inline asset-transfer-secured-agreement/README.md:8]: same comment as above.

File: asset-transfer-secured-agreement/README.md (hunk 1 above)
Comment (stephyee) [reply asset-transfer-secured-agreement/README.md]: 🛠️

File: asset-transfer-secured-agreement/README.md (hunk 2 above)
Comment (stephyee) [reply asset-transfer-secured-agreement/README.md]: 🛠️

File: asset-transfer-secured-agreement/README.md (hunk 3 above)
Comment (stephyee) [reply asset-transfer-secured-agreement/README.md]: 🛠️

**Comment by stephyee** [comment]
The images in the readme still uses marbles as the asset in the diagrams. In a follow-up we can generate new diagrams.

**Pull request merged**